src/applications/discharge-wizard/components/v2/resultsComponents/StepTwo.jsx

```diff
@@ -143,6 +143,63 @@ const StepTwo = ({ formResponses }) => {
     default:
   }
 
+  const renderBuddyStatements = () => {
+    return (
+      <li>
+        <strong>
+          Buddy statements (also called a Statement in Support of Claim or VA
+          Form 21-4138)
+        </strong>
+        <p>
+          These are statements from friends or colleagues who knew you during
+          your service, or individuals with direct knowledge of your service.
+        </p>
+        <p>
+          The person will need to fill out a Statement in Support of Claim (VA
+          Form 21-4138). They may include this information about you in their
+          statement:
+        </p>
+        <ul>
+          <li>Your achievements in the military</li>
+          <li>Positive relationships you formed in the military</li>
+          <li>Why your discharge is unjust or incorrect</li>
+          <li>Your positive contributions during your service</li>
+        </ul>
+        <p>
+          The writer should also say in their statement how they know this
+          information.{' '}
+        </p>
+        {/* Intentionally not using <va-link> per Platform Analytics team */}
+        <a
+          download
+          href="https://www.va.gov/find-forms/about-form-21-4138/"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Get VA Form 21-4138 (opens in a new tab)
```

Comment:
The text for this link includes `to download` in the ticket (screenshot below) - need to confirm here as well
<img width="258" alt="Screenshot 2024-11-15 at 3 08 56 PM" src="https://github.com/user-attachments/assets/cf5c769d-06fa-406f-9646-55cedcb9f7f4">